labeling/internal/github.go

```diff
@@ -0,0 +1,18 @@
+package internal
+
+import (
+	"path"
+
+	"github.com/CircleCI-Public/circleci-config/labeling/codebase"
+	"github.com/CircleCI-Public/circleci-config/labeling/labels"
+)
+
+var GithubActionRules = []labels.Rule{
+	func(c codebase.Codebase, ls labels.LabelSet) (label labels.Label, err error) {
+		label.Key = labels.CICDGithubActions
+		configPath, err := c.FindFile(".github/workflows/*yml")
```

Comment:
Should it be 
```suggestion
		configPath, err := c.FindFile(".github/workflows/*.yml")
```

Also should we look for `*.yaml` as well?